main.go

```diff
@@ -2,7 +2,7 @@ package main
 
 import (
 	"flag"
-	hnd "myip/handler"
+	hnd "github.com/leesutcliffe/myip/handler"
```

Comment:
Not required, see suggested changes

Reply:
reverted and added a `go.mod` into the root of the project